src/components/backup_clouds/nfs/backup_cloud.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+	<div class="backup-cloud-nfs">
+		<openwb-base-heading>
+			Einstellungen für NFS-Backup Cloud
+			<span class="small">(Modul: {{ $options.name }})</span>
+		</openwb-base-heading>
+		<openwb-base-text-input
+			title="Pfad zum NFS Share"
+			:model-value="configuration.nfs_share"
```

Comment:
```suggestion
			pattern="^([^/: ]+):(\/[^/: ]+)+$"
			:model-value="configuration.nfs_share"
```
Der Regex sollte eigentlich zu einem NFS Share passen.
`server:/dir1/dir2...`